hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BlockNodeConnectionConfig.java

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+package com.hedera.node.config.data;
+
+import com.hedera.node.config.NetworkProperty;
+import com.hedera.node.config.NodeProperty;
+import com.swirlds.config.api.ConfigData;
+import com.swirlds.config.api.ConfigProperty;
+
+/**
+ * Configuration for Connecting to Block Nodes.
+ * @param shutdownNodeOnNoBlockNodes whether to shut down the consensus node if there are no block node connections
+ * @param blockNodeConnectionFileDir directory to get the block node configuration file
+ * @param waitPeriodForActiveConnection the time in minutes to wait for an active connection
+ */
+@ConfigData("blockNode")
+public record BlockNodeConnectionConfig(
+        @ConfigProperty(defaultValue = "false") @NodeProperty boolean shutdownNodeOnNoBlockNodes,
+        @ConfigProperty(defaultValue = "/opt/hgcapp/data/config") @NodeProperty String blockNodeConnectionFileDir,
```

Comment:
For any paths defined in the config, we want to use relative paths as much as possible
```suggestion
        @ConfigProperty(defaultValue = "data/config") @NodeProperty String blockNodeConnectionFileDir,
```